pubg_python/querysets.py

```diff
@@ -31,10 +31,13 @@ def __getitem__(self, key):
         return Domain.instance({'data': self._data['data'][key]})
 
     @invalidates_cache
-    def get(self, id=None):
+    def get(self, id=None, season_id=None):
         # get can be a fetchy filter in the future
         if id:
             self.endpoint.path.segments.append(id)
+            if season_id:
+                self.endpoint.path.segments.append('seasons')
+                self.endpoint.path.segments.append(season_id)
```

Comment:
No need for this, the `seasons` URL will be automatically generated by the endpoint function and the `season_id` is the same as `id`.

Reply:
here I am trying to build this kind of endpoint url '/players/{id}/seasons/{seasonId}' by using the part of the URL already built for players. If requesting player data like this: ```api.players().get(id=pid, season_id=current_season_id)``` I add /seasons/{seasonId} to the path.

The current endpoint function does not support multiple segments from what I can tell and I chose to treat it as an edge case rather than changing the endpoint function because it integrates well with the .players() functionality and it is the only endpoint of this type (/a/{id_a}/b/{id_b}), I found.